src/web/components/ClickToView.tsx

```diff
@@ -137,64 +144,59 @@ export const ClickToView = ({
 				className={css`
 					width: 100%;
 					background: ${background.secondary};
-					border: 1px solid ${border.primary};
+					border: 1px solid ${border.secondary};
 					display: flex;
 					flex-direction: column;
 					justify-content: space-between;
-					padding: ${space[3]}px;
-					margin-bottom: 8px;
+					padding: 6px ${space[6]}px ${space[3]}px;
+					margin-bottom: ${space[3]}px;
 				`}
 				data-component={`click-to-view:${sourceDomain}`}
 			>
 				<div
 					className={css`
 						${roleHeadlineSize(role)}
-						margin-bottom: 8px;
+						margin-bottom: ${space[1]}px;
 					`}
 				>
 					{source
 						? `Allow ${source} content?`
 						: 'Allow content provided by a third party?'}
 				</div>
-				<div
+				<p
 					className={css`
 						${textSize}
-						a {
-							${textSize}
-						}
-						p {
-							margin-bottom: 8px;
+						margin-bottom: ${space[5]}px;
```

Comment:
Okay, I'm delving deep into nitpick terratory here, sorry! But if we set this spacing around the Button then that kind of communicates the intention better (eg. we want more space above the button) and protects us against if new paragraphs were to be added later.

What do you think?

Reply:
Sounds good :)